packages/astro/src/core/request.ts

```diff
@@ -92,3 +103,32 @@ export function createRequest({
 
 	return request;
 }
+
+/**
+ * Utility function that creates a new `Request` with a new URL from an old `Request`.
+ *
+ * @param newUrl The new `URL`
+ * @param oldRequest The old `Request`
+ */
+export function copyRequest(newUrl: URL, oldRequest: Request, isPrerendered: boolean): Request {
```

Comment:
Where is this version of the function used?

Reply:
Bad copy-paste. Good catch